vmdb/app/models/miq_request_workflow.rb

```diff
@@ -116,7 +116,7 @@ def init_from_dialog(values, userid)
 
     self.get_all_dialogs.keys.each do |d|                         # Go thru all dialogs
       self.get_all_fields(d).keys.each do |f|                     # Go thru all field
-        unless options[f].nil?
+        if !options[f].nil?
```

Comment:
do note that `false.present? == false`, so I don't think we want present? here.

@brandondunne please revisit this line and the other line to another PR